bin/img.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# are there images to optimize?
+fileschanged=$(git diff --diff-filter=ACM --name-only HEAD | grep ".png\|.jpg\|.svg" | wc -l)
```

Comment:
Not sure if this would be any easier w/ something like [**lint-staged**](https://github.com/okonet/lint-staged) and [**imagemin-cli**](https://github.com/imagemin/imagemin-cli). It would arguably be more cross-OS compatible, but probably wouldn't support things like the viewBox and high-res/low-res checks.